File: 프로그래머스/2/12899. 124 나라의 숫자/124 나라의 숫자.java
class Solution {
	public static String solution(int n) {
		String answer = "";
		String[] nums = {"4", "1", "2"};
		
		while(n>0) {
			answer = nums[n%3] + answer;
			n = (n-1)/3;
		}
		return answer;
	}
}

File: 프로그래머스/2/12899. 124 나라의 숫자/124 나라의 숫자.js
function solution(n) {
  let answer = "";
  let num_arr = [4, 1, 2];
  while (n > 0) {
    answer = num_arr[n % 3] + answer;
    // 이 뒤의 n을 어떻게 처리하냐...
    n = parseInt((n - 1) / 3);
  }
  return answer;
}